docs/service-filesystem-setup.md

```diff
@@ -0,0 +1,48 @@
+
+# Filesystem Setup
+
+The service expects the following filesystem structure:
+
+```ascii 
+ {asset_directory} 
+  ├── {project_folder} 
+  |    └──{asset_segment_1}
+  |        └── {asset_segment_2}
+  |              ├── xxx.info
+  |              ├── xxx.jpg.orig
+  |              └── xxx.jp2
+  ├── {project_folder} 
+  …    └──{asset_segment_1}
+           └── {asset_segment_2}
+                 ├── xxx.info
+                 ├── xxx.jpg.orig
+                 └── xxx.jp2               
+ {temp_directory}
+  ├── export
+  └── import
+```
+
+Here is a description of the folders:
+
+* `{asset_directory}`: the folder where the assets are stored
```

Comment:
This is not clear. Is this the `images` folder? If this are two mounted volumes, we know already `{asset_directory}` and `{temp_directory}` values right?